nilearn/input_data/tests/test_masker_validation.py

```diff
@@ -0,0 +1,85 @@
+from nose.tools import assert_true
+
+import nibabel
+import numpy as np
+
+from nilearn._utils.testing import assert_warns
+
+from sklearn.base import BaseEstimator
+from sklearn.externals.joblib import Memory
+
+from nilearn.input_data.masker_validation import check_embedded_nifti_masker
```

Comment:
You may want to group nilearn imports together, after the sklearn would make sense.